server/logs/logs.go

```diff
@@ -0,0 +1,24 @@
+/******************************************************************************
+ *
+ *  Description :
+ *    Package exposes info, warning and error loggers.
+ *
+ *****************************************************************************/
+package logs
+
+import (
+	"log"
+	"os"
+)
+
+var (
+    Info    *log.Logger
+    Warning *log.Logger
+    Error   *log.Logger
+)
+
+func Init() {
+	Info = log.New(os.Stdout, "I", log.LstdFlags | log.Lshortfile)
+	Warning = log.New(os.Stdout, "W", log.LstdFlags | log.Lshortfile)
```

Comment:
I would make these less verbose, way `Warn` and `Err`. We are going to write these lines a lot.

Reply:
Renamed.